doc/nrf/gs_installing.rst

```diff
@@ -177,26 +190,32 @@ Install west
 
 To manage the combination of repositories and versions, the |NCS| uses :ref:`Zephyr's west <zephyr:west>`.
 
-To install west, enter the following command:
+To install west, complete the following steps:
 
 .. tabs::
 
    .. group-tab:: Windows
 
+      Open a command window with administrator rights in the :file:`ncs` folder and enter the following command:
+
       .. parsed-literal::
          :class: highlight
 
          pip3 install west
 
    .. group-tab:: Linux
 
+      Open a terminal window with administrator rights in the :file:`ncs` folder and enter the following command:
```

Comment:
That's not how it works on Linux. You would use sudo with the command if you need administrator rights, and I really don't think you need those to install pip packages.

Reply:
Confirmed with one of the Matter team members.